src/components/gds-paragraph/gds-paragraph.tsx

```diff
@@ -14,10 +14,18 @@ export class GdsParagraph {
    * Size of the text.
    */
   @Prop() size: string = 'm'
+  /**
+   * Use to override p element's style.
+   */
+  @Prop() class: string
```

Comment:
Is it just me or is this going in a very dangerous direction? Personally I'd try every other approach before allowing custom classnames in any gds component. Then again this is quite a primitive component so perhaps it's justifiable.

I just hope next up we don't start applying custom font sizes and line heights through these classes.

Reply:
Yeah, I know. But this way I could remove the margin from that p element. See more comments below.